snap/snapcraft.yaml

```diff
@@ -8,20 +8,43 @@ summary: The easiest, most secure way to use WireGuard and 2FA
 description: |
   A secure network that just works Zero config VPN.
 
-  Installs on any device in minutes, manages firewall rules for you, and works
-  from anywhere.
+  Installs on any device in minutes, manages firewall rules for you, and works from anywhere.
 
-  ## Confinement
+  After installing this snap, visit the Tailscale documentation at https://tailscale.com/kb/1017/install to get started.
+
+  **Confinement**
 
   The snap is strictly confined, and this currently brings some limitations.
 
-  - `tailscale ssh` will not work.
-  - `ssh` over the tailscale network will not work if tailscale ssh mode is enabled (ie. if you have configured `tailscale set --ssh`). To fix this, disable tailscale ssh: `tailscale set --ssh=false`.
-  - `tailscale update` will not work: to update tailscale, please refresh the snap.
-  - `tailscale file cp` and `tailscale cert` will only work with file paths accessible to the snap, for example `/home/ubuntu/snap/tailscale/common/foo.txt`. See snap [data locations](https://snapcraft.io/docs/data-locations) for a full list.
-  - `tailscale drive share` will not work (the client command succeeds, the share is registered, but you will see errors in tailscaled logs and the share will not be accessible).
+  * `tailscale ssh` will not work.
```

Comment:
I think these information are also useful in `README.md` (at least mentioning not all features are available in the snap). To avoid duplication, I suggest add a link in the `README.md` pointing to snapcraft listing (or some thing similar)

Reply:
Good point thanks, I added a note about the user vs developer docs and a link to the snap store listing. Note that the snap store page is a 404 right now - I removed the old releases to tidy things up. It will be there again once we get approval to publish the new snap. :)

Reply:
Maybe a known issues doc/page? Since there are some issue required to permission. A link on README can be useful.

Reply:
That's a good idea - I can expand on them and add more information later (maybe during testing this pulse).

Reply:
Thanks